content/20.treatments.md

```diff
@@ -318,6 +318,8 @@ _In vitro_ evidence suggests that HCQ may be an effective therapeutic against SA
 As recently discussed [@doi:10.1093/jac/dkaa114], because the 90% effective concentration (EC~90~) of CQ in Vero E6 cells (6.90 μM) can be achieved in and tolerated by rheumatoid arthritis patients, it may also be possible to achieve the effective concentration in COVID-19 patients.
 Additionally, HCQ was found to be effective in HIV [@doi:10/cq2hx9] and chronic Hepatitis C [@doi:10.1002/jmv.24575] patients.
 Therefore, initially it was thought that CQ/HCQ would be effective against SARS-COV-2.
+There are few reasons why HCQ was used more commonly than CQ.
+The hypotheses were that HCQ could alleviate the development of COVID-19 through mitigation of CD154 in T cells, could have the same effect as an antiviral like CQ pre- and post-infection, is cheaper, and has less side effects than CQ
```

Comment:
I think you wanted this sentence struck before (#340 or #338)-- did you want to workshop it in this version?

Reply:
yes

Reply:
@soumitagh OK! I would suggest a little change because my understanding is that HCQ is just the synthetic version of CQ:
```suggestion
CQ and HCQ have both been found to inhibit the expression of CD154 in T-cells and to reduce toll-like receptor signaling that leads to the production of pro-inflammatory cytokines [@doi:10.1038/s41584-020-0372-x], but HCQ offers the advantages of being cheaper and having fewer side effects than CQ.
```